kitchen-ec2.gemspec

```diff
@@ -18,8 +18,11 @@ Gem::Specification.new do |gem|
   gem.test_files    = gem.files.grep(%r{^(test|spec|features)/})
   gem.require_paths = ["lib"]
 
-  gem.add_dependency 'test-kitchen', '~> 1.0'
-  gem.add_dependency 'fog'
+  gem.add_dependency "test-kitchen", "= 1.4.0.rc.1"
+  gem.add_dependency 'excon'
+  gem.add_dependency 'multi_json'
```

Comment:
Is there a reason for multi_json other than older Ruby support? Test Kitchen only works on Ruby 1.9 and up which ships with a 'json' in stdlib. I may be missing a hidden dep in here though…